references/optical_flow/train.py

```diff
@@ -243,7 +237,8 @@ def main(args):
     scheduler = torch.optim.lr_scheduler.OneCycleLR(
         optimizer=optimizer,
         max_lr=args.lr,
-        total_steps=args.num_steps + 100,
+        epochs=args.epochs + 1,
```

Comment:
We had a +100 before.
Without this +1, I'm consistently seeing some weird errors when training on Kitti for 2 epochs:

<details>

```
Traceback (most recent call last):
  File "/fsx/users/nicolashug/work/vision/references/optical_flow/train.py", line 345, in <module>
    main(args)
  File "/fsx/users/nicolashug/work/vision/references/optical_flow/train.py", line 254, in main
    train_one_epoch(
  File "/fsx/users/nicolashug/work/vision/references/optical_flow/train.py", line 173, in train_one_epoch
    for data_blob in logger.log_every(train_loader):
  File "/fsx/users/nicolashug/work/vision/references/optical_flow/utils.py", line 123, in log_every
    for obj in iterable:
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 521, in __next__
    data = self._next_data()
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1203, in _next_data
    return self._process_data(data)
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1229, in _process_data
    data.reraise()
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/_utils.py", line 438, in reraise
    raise exception
RuntimeError: Caught RuntimeError in DataLoader worker process 9.
Original Traceback (most recent call last):
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/_utils/worker.py", line 287, in _worker_loop
    data = fetcher.fetch(index)
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 52, in fetch
    return self.collate_fn(data)
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/_utils/collate.py", line 100, in default_collate
    return [default_collate(samples) for samples in transposed]  # Backwards compatibility.
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/_utils/collate.py", line 100, in <listcomp>
    return [default_collate(samples) for samples in transposed]  # Backwards compatibility.
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/utils/data/_utils/collate.py", line 64, in default_collate
    storage = elem.storage()._new_shared(numel)
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/storage.py", line 486, in _new_shared
    untyped_storage = module.UntypedStorage._new_shared(size * cls().element_size())
  File "/fsx/users/nicolashug/conda/envs/raft/lib/python3.9/site-packages/torch/storage.py", line 173, in _new_shared
    return cls._new_using_fd(size)
RuntimeError: unable to open shared memory object </torch_24843_0> in read-write mode: File exists (17)
```

</details>

Reply:
This is really concerning but I don't know what could cause this. @fmassa do you have any thoughts on what this could be?

Reply:
I'm unable to reproduce the error today. I'm suspecting that this is related to the state of my env at the time. I do observe some rare ddp failures from time to time, so I'm hoping that this was just one of these one-off events.

I ran the training with `epochs + 1` and got good epe resuts on C+T. I will remove the `+ 1` and train again. If I still get good resuts, and if I still can't reproduce, I think it'll be safe to merge.